src/EditorFeatures/CSharpTest2/Recommendations/NotKeywordRecommenderTests.cs

```diff
@@ -0,0 +1,183 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Recommendations
+{
+    public class NotKeywordRecommenderTests : KeywordRecommenderTests
+    {
+        private const string InitializeObjectE = @"object e = new object();
+";
+
+#if !CODE_STYLE
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestAfterIsKeyword()
+        {
+            await VerifyKeywordAsync(AddInsideMethod(InitializeObjectE +
+@"if (e is $$"));
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestAfterNotKeyword()
+        {
+            await VerifyKeywordAsync(AddInsideMethod(InitializeObjectE +
+@"if (e is not $$"));
```

Comment:
interesting.  i guess this would be legal.  though we might consider suppressing here.  but very minor and totally fine like this. :)